tools/container/toolkit/toolkit.go

```diff
@@ -793,3 +791,19 @@ func generateCDISpec(opts *Options, nvidiaCDIHookPath string) error {
 
 	return nil
 }
+
+// getNvidiaContainerRuntimeList returns a consolidated list of low-level runtimes to be used by the toolkit
+func getNvidiaContainerRuntimeList(cfgToml *config.Toml, containerRuntimeBinaries []string) []string {
+	var ctkRuntimes []string
+	nvidiaCTKConfig, err := cfgToml.Config()
+	if err == nil {
+		if len(containerRuntimeBinaries) > 0 {
+			ctkRuntimes = append(ctkRuntimes, containerRuntimeBinaries...)
+		}
+		defaultCfgRuntimes := nvidiaCTKConfig.NVIDIAContainerRuntimeConfig.Runtimes
+		ctkRuntimes = append(ctkRuntimes, defaultCfgRuntimes...)
+	} else {
+		log.Warningf("could not get nvidia container toolkit config: %v", err)
+	}
+	return ctkRuntimes
+}
```

Comment:
I think we should remove this logic. It makes it more complex to reason about what's happening.

Reply:
Without this, we won't be able to prepend the `nvidia-container-runtime.runtimes` list with the extracted low level runtime binary paths.

Reply:
I don't think it's needed to prepend the values in this case. The config comes from the toolkit itself and there isn't a way for the user to set this so if we really want to prepend, we can just add `docker-runc`, `runc`, `crun` to the end of the `opts.ContainerRuntimeRuntimes` where we update it.

Reply:
Ok, I see your point now. Making the change

Reply:
Done. Please check now